src/style.css

```
@@ -1,3 +1,75 @@
-body {
+:root {
+  --my-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
```

Instead of relying on the asterisk selector, it's generally recommended to selectively target the elements or classes that require the specific styles. This provides better control, improves performance, and helps avoid unintended side effects or conflicts with other styles in the project.
Move the font-family to the body selector
```suggestion
* {
  box-sizing: border-box;
}
```